test/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+#
+# this file is used if the developer has opted not to use Hunter cmake extensions
```

Comment:
What's wrong with plain CMake? This adds a lot of indirection, and I'm not seeing any advantage right now.

Reply:
Try building a project for OSX, linux, windows, ISO and android. You'll soon start to appreciate not having to pre-build all your dependencies for every platform, keep those dependencies up to date, and remember where you built them.

This is what hunter does for you. It's the 80% of cmake that's missing.

Reply:
Remove ISO from that list, add QNX and WinCE and you're describing the list of platforms that I've had to target with CMake for a single (large) project. Pre-building is effectively a caching strategy: it can be useful (during development it certainly is), but it's very hard to get right and the benefits are non-existent for small projects.

I do thank you for pointing it out to me, because it may be useful at work.

This however, is a header-only library, so there's no external dependency to be built. I.e. the tests are the only build artifact and those only use Boost's header-only part too.

Reply:
WinCE and QNX are covered with hunter by using the toolchain files that have been conveniently supplied by ruslo. the project on github is ruslo/polly.

ISO == iOS, my apologies.

Let's re-discuss when you have come to get to know hunter.

The author is not english so the documentation can take some decoding. Feel free to ping me if you have questions.